client/app/common/hero/hero.js

```diff
@@ -6,6 +6,8 @@ let heroModule = angular.module('hero', [
   uiRouter
 ])
 
-.component('hero', heroComponent);
+.component('hero', heroComponent)
+  
+.name;
 
 export default heroModule;
```

Comment:
@blakedietz I think you right, it would be better to stick to @toddmotto guidelines.